public/scripts/javascript.js

```diff
@@ -1,4 +1,25 @@
-document.getElementById("logoutButton").addEventListener("click", function() {
+document.getElementById("logoutButtonNavBar").addEventListener("click", function() {
+    fetch("/logout", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+      }),
```

Comment:
Do we need to send anything to this endpoint?

Reply:
@eoinmcbennett I did not, this code has been removed.